pkg/mkimage-raw-efi/make-raw

```diff
@@ -203,6 +236,10 @@ do_vfat() {
 
 do_conf() {
     do_vfat $1 13307e62-cd9c-4920-8f9b-91b45828b798
+    if [ -z "$RANDOM_DISK_UUIDS" ]; then
+        local NUM_PART=$(( "$PART_OFFSET" + 4 ))
```

Comment:
I can see `/make-raw: eval: line 240: arithmetic syntax error` here, probably it should be `local NUM_PART=$(( PART_OFFSET + 4 ))`

Reply:
I got a yetus complaint for that line when PART_OFFSET was not quoted.
Sure you got the error for this line?

Reply:
You're right. I made the references to PART_OFFSET be consistent since it was mixing $PART_OFFSET and PART_OFFSET in different places.